sdk/state/state.go

```diff
@@ -35,6 +35,24 @@ type Channel struct {
 
 	latestAuthorizedCloseAgreement   CloseAgreement
 	latestUnauthorizedCloseAgreement CloseAgreement
+
+	closeState CloseState
+}
+
+type CloseState int
```

Comment:
a CloseState field is a simplification for now for this PR. It goes against the pattern of the other code in this sdk of getting the channel state from the latest agreements. I couldn't think of a good way of signaling a close state for now, so simplifying for now and going to do that in this ticket: https://github.com/stellar/experimental-payment-channels/issues/233

Reply:
I refactored it in this commit: https://github.com/stellar/experimental-payment-channels/pull/232/commits/85ceac50beb42dab6dd4ec55c9439789b03b7a58

now the CloseState is inferred from comparing the network sequence number to what we expect it to be. This more goes with the pattern we're currently using where the channel state is inferred from Agreements and data from the network